.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "salt/journal-cms/api-dummy"]
```

Comment:
this file should not be there: formulas do not have submodules because dependent formulas will be checked out by the master-server for them